Packs/GoogleThreatIntelligence/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,163 @@
+#### Integrations
+
+##### New: Google Threat Intelligence - ASM Issues
+
+- This integration allows the creation of incidents based on ASM Issues from Google Threat Intelligence.
```

Comment:
The release note description does not start with an approved template phrase. Please rephrase this line and line 15 to use one of the standard prefixes.